Dokumentationen/requirements/survey_for_essence.adoc

```diff
@@ -0,0 +1,70 @@
+= Umfrage zur Zufriedenheit der Gruppe in Bezug der Projektarbeit
+
```

Comment:
Vlt noch eine kurze Erläuterung, in welchem Zeitraum und aus welchem Anlass die Umfrage durchgeführt wurde?